### DISCUSSION TITLE
PaddleOCR c# not work in VM windows Server 2016

### DISCUSSION THREAD

**Discussion by emranallan** [PaddlePaddle/PaddleOCR#13615] [Q&A]
### Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.

- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.

- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### Bug

my code in C# work in my PC but not work in VM Server 2016
give me this error
'iisexpress.exe' (CoreCLR: clrhost): Loaded 'C:\Program Files\dotnet\shared\Microsoft.NETCore.App\8.0.7\System.Runtime.InteropServices.RuntimeInformation.dll'. 
The program '[12136] iisexpress.exe' has exited with code 3221225501 (0xc000001d) 'Illegal Instruction'.

### Environment

VM server 2016

### Minimal Reproducible Example

   FullOcrModel model = LocalFullModels.EnglishV3;
   PaddleOcrAll all = new PaddleOcrAll(Globals.model, PaddleDevice.Mkldnn())
   {
       AllowRotateDetection = false,
       Enable180Classification = false,
   };    

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by GreatV** [comment]
PaddleOCR doesn't provide C# code, so we can't help you with that.

**Comment by n0099** [comment]
https://github.com/sdcb/PaddleSharp/issues/106
